chunk/tag.go

```diff
@@ -58,13 +65,6 @@ type Tag struct {
 	ctx      context.Context  // tracing context
 	span     opentracing.Span // tracing root span
 	spanOnce sync.Once        // make sure we close root span only once
```

Comment:
I guess I broke the alignment with the introduction of `spanOnce` a few PRs back.